src/App.tsx

```diff
@@ -1,18 +1,45 @@
 import React from 'react';
 
 import './App.scss';
+import { Tabs } from './components/Tabs';
 
 const tabs: Tab[] = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+type State = {
+  selectedTab: Tab;
+};
+
+class App extends React.Component<{}, State> {
+  state = {
+    selectedTab: tabs[0],
+  };
+
+  changeTab = (newSelectedTab: Tab) => {
+    this.setState({
+      selectedTab: newSelectedTab,
+    });
+  };
+
+  render() {
+    const { selectedTab } = this.state;
+
+    return (
+      <div className="App">
+        <h1 className="App__title">{`Selected tab is ${this.state.selectedTab.title}`}</h1>
```

Comment:
```suggestion
        <h1 className="App__title">{`Selected tab is ${selectedTab.title}`}</h1>
```
you have destructuring already

Reply:
Поспешил, и не убрал так сказать "черновой вариант".